web_external/Front/frontPage.pug

```diff
@@ -1,110 +1,50 @@
-            |  includes 557 cutaneous melanoctic lesions from adults (&gt;=18 years
-            |  of age) with either a definitive histopathologic diagnosis (for
-            |  excised lesions), or a documented history of being clinically
-            |  benign through review of serial clinical images acquired at least
-            |  6 months apart. Lesions were excluded from the dataset if there
-            |  was equivocal histopathology, unacceptable image quality, or if
-            |  the lesions were located on the face, acral or mucosal sites.
-          h4 ISIC_UDA-2:
-          p
-            | This dataset contains 44 high quality lesion images of melanomas
-            |  and benign lesions. Along with the image data, metadata is
-            |  available related to patient age, diagnosis, gender, and anatomic
-            |  location. For this dataset, four expert dermoscopists independently
-            |  reviewed and annotated all the lesions images for dermoscopic
-            |  patterns and structures.
-          h4 ISIC_MSK-1:
-          p
-            | The lesions included in this dataset were acquired from a single
-            |  pigmented lesion clinic. This dataset only includes lesions that
-            |  were excised. Relevant clinical metadata such as diagnosis,
-            |  anatomic location, gender age and clinical impression of the
-            |  lesion are included with each lesion image.
-          h4 ISIC_MSK-2:
-          p
-            | Benign and malignant skin lesions. Biopsy-confirmed melanocytic
-            |  and non-melanocytic lesions.
-          h4 ISIC_SONIC-1:
-          p
-            | Moles in children. Benign melanocytic lesions from a pediatric
-            |  population. The benign nature of the lesions is based on clinical
-            |  assessment and/or no change on serial imaging.
+.isic-frontpage
+  .container
+    .isic-frontpage-banner.row
+      .col-lg-12
+        h1 Welcome to ISIC
+        h2 The International Skin Imaging Collaboration
+    .isic-frontpage-panels.row
+      .col-lg-12
+        .row
+          .col-lg-3
+            .isic-frontpage-about.isic-frontpage-panel
+              i.icon-info
+              h3 About ISIC
+              span Learn about the ISIC Project and our goals to advance melanoma research
+          .col-lg-3
+            .isic-frontpage-images.isic-frontpage-panel
+              i.icon-picture
+              h3 View Gallery
+              span Explore collections of high quality image data sets
+          .col-lg-3
+            .isic-frontpage-challenges.isic-frontpage-panel
+              i.icon-award
+              h3 Machine Learning Challenges
+              span Participate in open competitions and review past challenges
+          .col-lg-3
+            .isic-frontpage-upload.isic-frontpage-panel
+              i.icon-upload-cloud
+              h3 Upload Data
+              span Contribute images and data to the ISIC Archive
```

Comment:
In the menu we show this only for users who are logged in:
https://github.com/ImageMarkup/isic-archive/blob/76d91500b2eb71c1249554b21ab85705f5659cf6/web_external/layout/layoutHeader.pug#L88

Maybe this panel could be greyed out or otherwise disabled?